### PR TITLE
RIA-3006 fee payment before appeal submit

### PR DESCRIPTION
**Pull request by Lee365**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RIA-3237

### Change description ###
RIA-3006 fee payment before appeal submit

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

File: e2e/features/stepDefinitions/service/overview.steps.ts [hunk 1]
```diff
@@ -39,6 +39,46 @@ Then(/^I should (see|not see) the case details$/, async function (seeOrNotSee) {
     }
 });
 
+Then(/^I should (see|not see) the case details with (DC|RP) type$/, async function (seeOrNotSee, DeprivationOrRevocation) {
```
Comment (jkumar-moj) [inline e2e/features/stepDefinitions/service/overview.steps.ts:42]: Could we not expand DC/RP? As someone new or some non dev reading through the executed steps I can not determine the involved action here. After looking at I could still raise a question is RP - Revocation of Protection or Refusal of Protection. What's your opinion @Lee365?

Comment (Lee365) [reply e2e/features/stepDefinitions/service/overview.steps.ts]: Agreed - expanded the DC/RP to be clear.

Comment (jkumar-moj) [reply e2e/features/stepDefinitions/service/overview.steps.ts]: Thanks @Lee365

File: e2e/flows/submit-hearing-requirements.flow.ts [hunk 2]
```diff
@@ -98,7 +98,7 @@ export class SubmitHearingRequirementsFlow {
         await this.ccdFormPage.click('Continue');
 
         await this.ccdFormPage.setFieldValue(
-            'Does the appellant have any physical or mental health issues that may impact them during the hearing?',
+            'Does the appellant have any physical or mental health issues that may impact them',
```
Comment (Srihari-Nakkala) [inline e2e/flows/submit-hearing-requirements.flow.ts:101]: @Lee365 Any Reason we Modified the Content here.

Comment (Lee365) [reply e2e/flows/submit-hearing-requirements.flow.ts]: Reverted this change after rebase from master.

**Pull request closed without merging**